org-roam-bibtex.el

```diff
@@ -490,7 +490,7 @@ calling any Org-roam functions."
               (let ((org-roam-capture--context 'ref)
                     (org-roam-capture--info (list (cons 'title title)
                                                   (cons 'ref citekey-formatted)
-                                                  (cons 'slug (org-roam--title-to-slug citekey)))))
```

Comment:
Please, revert this change.

Reply:
Reverted in 4df39a7cddc2e7ba19154fa24db39f9fe3349292.